lib/fluent/plugin/filter_record_transformer.rb

```diff
@@ -92,17 +92,45 @@ def filter_stream(tag, es)
 
     private
 
+    def parse_value(value_str)
+      if /^[\[{]/.match(value_str)
+        JSON.parse(value_str)
+      else
+        value_str
+      end
+    rescue => e
+      log.warn "failed to parse #{value_str}", :error_class => e.class, :error => e.message
```

Comment:
This message is not good for users.
`"failed to parse #{value_str}" as json. Assume #{value_str} is string parameter` or something is good.